testing/trino-product-tests-launcher/src/main/resources/docker/presto-product-tests/conf/environment/singlenode-spark-iceberg/hive-site-overrides.xml

```diff
@@ -0,0 +1,9 @@
+<configuration>
+
+    <property>
+        <!-- Required to enable full schema evolution. See https://github.com/apache/iceberg/issues/1092#issuecomment-638432848 -->
```

Comment:
Q would this be also required if we test the same behavior for hive connector? (with name-based mapping)

Reply:
This i do not know, but sounds possible.
Note however, that for Iceberg, metastore is not authoritative on table schema, so can be seen as an obstacle only. 
As an alternative, we could have a test that runs without HMS, eg with lightweight JDBC-based catalog for Iceberg, if Trino Iceberg supported that. Disabling the metastore check sounded like much lower hanging fruit.

For Hive, we should understand first whether such "arbitrary" schema evolution is what users actually do. If they do, we may choose to support that: add such configuration along with tests. If they do not, we may choose to say 🤷‍♂️ .

However, using this could potentially help us avoid reducing test coverage for other Hive versions as I did in cdafc868e62827ef722732081bc0edfa1195e4dd.